recipe/meta.yaml

```
@@ -13,13 +13,15 @@ source:
 build:
   number: 0
   noarch: generic
+  skip: true # [not ((linux and x86_64) or win)]
```

```suggestion
  skip: true # [not (osx or s390x)]
```
Because we still want to test linux-aarch64